pdk/1.0/modules/ROOT/pages/policies-pdk-apply-policies.adoc

```diff
@@ -12,19 +12,74 @@ To learn more about defining configuration parameters, see xref:policies-pdk-cre
 
 == Before You Begin
 
-* xref:policies-pdk-publish-policies.adoc[Upload your Custom Policy to Exchange]
+* xref:policies-pdk-compile-policies.adoc[Compile your custom policy].
+* If applying your policy to Flex Gateway running in Connected Mode, xref:policies-pdk-publish-policies.adoc[upload your custom policy to exchange].
 
-== Connected Mode
+== Apply Policies in Connected Mode
 
 When you apply a custom policy in Connected Mode, the configuration parameters and metadata defined in `gcl.yaml` appear in the API Manager UI.
 
 To apply a custom policy in Connected Mode, see xref:gateway::policies-included-apply.adoc[Apply a Policy in Connected Mode]. 
 
-== Local Mode
+=== Update Policies in Connected Mode
+
+To update a custom policy in API Manager, you can either:
+
+* Delete the policy, and then re-apply the latest version of the policy. This automatically updates the policies binary file.
```

Comment:
I would put this option as the second. And I would suggest that it is only for DEV policies (the policy versions uploaded with `make publish`). For policies applied on production APIs it is not desirable to, even for a short period, leave the API unprotected.

Actually... if we don't want to over explain, I would remove this alternative and simply instruct with the second.